src/main/java/com/bgsoftware/superiorskyblock/service/placeholders/PlaceholdersServiceImpl.java

```diff
@@ -182,6 +183,7 @@ public class PlaceholdersServiceImpl implements PlaceholdersService {
                     .put("bank_limit", (island, superiorPlayer) -> Formatters.NUMBER_FORMATTER.format(island.getBankLimit()))
                     .put("bank_limit_format", (island, superiorPlayer) ->
                             Formatters.FANCY_NUMBER_FORMATTER.format(island.getBankLimit(), superiorPlayer.getUserLocale()))
+                    .put("island_coop_online", (island, superiorPlayer) -> String.valueOf(island.getCoopPlayers().stream().filter(SuperiorPlayer::isOnline).count()))
```

Comment:
Because it conflicts with the COOP pattern, and the term "online" for coop players is redundant, let's maybe rename this to `coops`?